aiohttp/test_utils.py

```diff
@@ -568,8 +568,8 @@ def make_mocked_request(method, path, headers=None, *,
     :param path: str, The URL including *PATH INFO* without the host or scheme
     :type path: str
 
-    :param headers: CIMultiDict with all request headers
-    :type headers: multidict.CIMultiDict
+    :param headers: dict with all request headers
+    :type headers: dict
```

Comment:
it can be multidict also. Technically the parameter is something acceptable by CIMultiDict ctor. Say, a list of pairs is also  working.